src/site/layouts/health_care_region_detail_page.drupal.liquid

```diff
@@ -126,9 +126,9 @@
               {% endif %}
             {% endfor %}
 
-            {% if fieldRelatedLinks and fieldRelatedLinks.entity.fieldVaParagraphs and fieldRelatedLinks.entity.fieldVaParagraphs.0.entity.fieldLink %}
+            {% if fieldRelatedLinks.0.entity.fieldVaParagraphs.0.entity.fieldLink %}
```

Comment:
Looks like some unnecessary nesting of data might be going on here.
Could this be addressed by changing the graphQL query instead?

Reply:
I don’t think so - this relates to the way paragraph entities are attached to content types in Drupal. In this case, our content model has a parent content type attached to a paragraph, that is attached to another paragraph.